packages/spacecat-shared-data-access/test/unit/models/importer/import-job.test.js

```diff
@@ -51,6 +51,63 @@ describe('ImportJob Model tests', () => {
       expect(() => createImportJob({ ...validImportJob, hashedApiKey: 123 })).to.throw('Invalid API key: 123');
     });
 
+    it('verify supported options', () => {
+      const job = createImportJob({
+        ...validImportJob,
+        options: {
+          [ImportOptions.ENABLE_JAVASCRIPT]: true,
+          [ImportOptions.PAGE_LOAD_TIMEOUT]: 1000,
+        },
+      });
+      expect(job.getOptions()).to.deep.equal({
+        [ImportOptions.ENABLE_JAVASCRIPT]: true,
+        [ImportOptions.PAGE_LOAD_TIMEOUT]: 1000,
+      });
+    });
+
+    /**
+     * There is an expectation that values coming into the import job ran through the
+     * body-data-wrapper's coerce logic, converting the string values into their potential data
+     * types like booleans, integers.
+     */
+    /* eslint-disable max-len */
+    it('test option data types', () => {
+      // Enable Javascript checks
+      expect(() => createImportJob({ ...validImportJob, options: { [ImportOptions.ENABLE_JAVASCRIPT]: true } })).to.not.throw();
```

Comment:
If the method in question threw an error, wouldn't that fail the test?

Reply:
I see where you're going.  To not have .to.not.throw doesn't seem to catch ones eye as what the test is doing.  Just to do an expect with nothing seems kind of empty.